WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1518,6 +1518,10 @@ private boolean handleBackPressed() {
         return true;
     }
 
+    public void onEditorBackPressed() {
+        getOnBackPressedDispatcher().onBackPressed();
+    }
+
```

Comment:
I couldn't find a call to this method across the code base, so I wonder if we need it.

Reply:
Nice catch! This was the first approach I took 😅 

This is handled by `onBackHandlerButton`. I'll remove it, thanks!

Reply:
Removed in https://github.com/wordpress-mobile/WordPress-Android/pull/19828/commits/96601500e2335cb70d0d79ff75bfdf78e507ce8b